src/components/Banner.js

```diff
@@ -45,53 +45,57 @@ const defaultProps = {
     shouldRenderHTML: false,
     shouldShowIcon: false,
     shouldShowCloseButton: false,
-    onClose: () => {},
-    onPress: () => {},
+    onClose: undefined,
+    onPress: undefined,
```

Comment:
I have a concern about this potentially causing an error on click, but I cannot easily test this because this pull request is from a branch on your forked repo. I don't believe this is allowed but someone correct me if I'm wrong

Reply:
@arosiclair  You can test this by implementing these changes locally, it is only a few lines. From my testing I can say, it does not cause any issues on a click as it acts as if you didn't provide the `onPress` prop and that's ok because `onPress` is an optional prop

Reply:
Nevermind this I was wrong!